Mathlib/Combinatorics/SimpleGraph/Basic.lean

```diff
@@ -550,6 +553,10 @@ instance fintypeEdgeSetSdiff [DecidableEq V] [Fintype G₁.edgeSet] [Fintype G
   rw [edgeSet_sdiff]
   exact Set.fintypeDiff _ _
 
+theorem card_top_edgeSet [DecidableEq V] [Fintype V] :
+    Fintype.card (⊤ : SimpleGraph V).edgeSet = (Fintype.card V).choose 2 := by
+  simp only [edgeSet_top, Set.coe_setOf, Sym2.card_subtype_not_diag]
```

Comment:
I think it's generally better to avoid a squeezed terminal simp unless it's necessary for perf/looping reasons.
```suggestion
  simp [edgeSet_top]
```